src/procedures/the-cat-api/the-cat-api.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * The Cats Service provides access to the Cat API data from https://api.thecatapi.com/v1/.
+ *
+ * @service
+ * @category Society
+ */
+
+const { TheCatApiKey } = require("../utils/api-key");
+const { registerTypes, CAT_BREEDS } = require("./types");
+const ApiConsumer = require("../utils/api-consumer");
+registerTypes();
+
+// Cats API Url
+const catApiUrl = "https://api.thecatapi.com/v1/images/search";
+
+const TheCatApi = new ApiConsumer("TheCatApi", catApiUrl, {
+  cache: { ttl: 1 },
+});
+ApiConsumer.setRequiredApiKey(TheCatApi, TheCatApiKey);
+
+/**
+ * Get random cat image.
+ * @param {CatBreeds=} catBreed cat breed supported by API.
```

Comment:
Usually for default values we add a description of what the default is/does. I'm guessing the default here does no breed-based filtering, so you could describe it like "A specific breed to search for. If omitted, no breed-based filtering is performed."